app/Resources/views/legislatives/contact.html.twig

```diff
@@ -0,0 +1,89 @@
+{% extends 'base.html.twig' %}
+
+{% block page_title 'Contact | Législatives' %}
+{% block canonical_url url('app_legislative_candidates_platform_contact') %}
+
+{% block content %}
+    <article class="l__wrapper--slim b__nudge--bottom-huge l__col l__col--center">
+        <h1>Vous avez une question ?</h1>
+        <h2>Contactez-nous</h2>
+        <div class="register__form">
+            <form action="{{ path('app_legislative_candidates_platform_contact') }}" method="post" novalidate="novalidate" id="legislatives_campaign_contact_form">
```

Comment:
And also because it doesn't provide enough flexibility to add the markup I want in the <form> tag.